### PR TITLE
Fm/features/calendar events queries

### PR DESCRIPTION
**Pull request by frankmangone**

### CODE REVIEW

File: src/entities/calendarEvent/calendarEvent.inputs.ts [hunk 1]
```diff
@@ -32,15 +34,15 @@ export class CreateCalendarEventInput {
 }
 
 @InputType()
-export class ListCalendarEventInput {
+export class ListCalendarEventsInput {
   @Field(() => ID, { nullable: true })
-  _id?: Types.ObjectId
+  calendar?: Types.ObjectId
 
   @Field(() => String, { nullable: true })
```
Comment (gastonmorixe) [inline src/entities/calendarEvent/calendarEvent.inputs.ts:41]: @frankmangone shouldn't this be a graphql date / date time type?

File: src/entities/calendarEvent/calendarEvent.inputs.ts [hunk 2]
```diff
@@ -32,15 +34,15 @@ export class CreateCalendarEventInput {
...
-  title?: string
+  rangeStart?: string
 
-  @Field(() => ID, { nullable: true })
-  calendar: Types.ObjectId
+  @Field(() => String, { nullable: true })
```
Comment (gastonmorixe) [inline src/entities/calendarEvent/calendarEvent.inputs.ts:44]: same above

File: src/entities/careerSubject/careerSubject.service.ts [hunk 3]
```diff
@@ -47,10 +47,10 @@ export class CareerSubjectService {
 
   // Pagination
 
-  async list(filters?: any, paginationArgs?: PaginationArgs) {
-    const { first, after, before } = paginationArgs
+  async list(filters?: any, connectionArgs?: ConnectionArgs) {
+    const { first, after, before } = connectionArgs
```
Comment (gastonmorixe) [inline src/entities/careerSubject/careerSubject.service.ts:51]: cool 😎

**Review by gastonmorixe — CHANGES_REQUESTED** [review]
Great work @frankmangone  :) few comments then LGTM

File: src/entities/calendarEvent/calendarEvent.inputs.ts (hunk 1 above)
Comment (frankmangone) [reply src/entities/calendarEvent/calendarEvent.inputs.ts:41]: Oh, no, it's as it should be. Think of what you'd your query would be like. You'd want query, for example, the CalendarEvents belonging to a Calendar. Maybe we could make this more complex in the future, but for now, it should suffice, along with the ability to list CalendarEvents followed by Users

File: src/entities/calendarEvent/calendarEvent.inputs.ts (hunk 2 above)
Comment (frankmangone) [reply src/entities/calendarEvent/calendarEvent.inputs.ts:44]: same as above!

File: src/entities/careerSubject/careerSubject.service.ts (hunk 3 above)
Comment (frankmangone) [reply src/entities/careerSubject/careerSubject.service.ts:51]: Thanks!

File: src/entities/calendarEvent/calendarEvent.inputs.ts (hunk 2 above)
Comment (frankmangone) [reply src/entities/calendarEvent/calendarEvent.inputs.ts:44]: Nevermind, fixed this.

**Comment by frankmangone** [comment]
Closes issue #26

**Pull request merged**